src/ai/backend/manager/api/manager.py

```diff
@@ -245,6 +245,25 @@ async def perform_scheduler_ops(request: web.Request, params: Any) -> web.Respon
     return web.Response(status=204)
 
 
+@superadmin_required
+async def scheduler_healthcheck(request: web.Request) -> web.Response:
+    root_ctx: RootContext = request.app["_root.context"]
+    raw_date = request.headers.get("Date")
+    if not raw_date:
+        raw_date = request.headers.get("X-BackendAI-Date", request.headers.get("X-Sorna-Date"))
+    if not raw_date:
+        raise GenericBadRequest
+    data = cast(dict, await root_ctx.shared_config.etcd.get_prefix("manager/scheduler"))
+    await root_ctx.event_producer.produce_event(DoScheduleEvent())
```

Comment:
How about divide this API into scheduler_trigger API and health check API?

Reply:
Thanks!
I decided trigger API and health check API.
And added prepare, scale_services trigger too. 👍